models/pathology_tumor_detection/docs/README.md

```diff
@@ -126,7 +130,7 @@ python -m monai.bundle run --config_file configs/train.json --dataset_dir <actua
 #### Override the `train` config to execute multi-GPU training
 
 ```
-torchrun --standalone --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']"
+torchrun --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']"
 ```
 
 Please note that the distributed training-related options depend on the actual running environment; thus, users may need to remove `--standalone`, modify `--nnodes`, or do some other necessary changes according to the machine used. For more details, please refer to [pytorch's official tutorial](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).
```

Comment:
Looks we can remove "to remove `--standalone`" here?

Reply:
thanks, I removed this change.